src/triples-for.js

```diff
@@ -220,23 +210,75 @@ export function triplesForScope(scopeUri, options) {
         ),
       };
     });
+}
+
+function getNodeShapeConstraints(constraints, scopeOptions) {
+  const { store, formGraph } = scopeOptions;
+  return toPathConstraints(
+    constraints.filter((constraint) =>
+      store.any(constraint.object, RDF("type"), SHACL("NodeShape"), formGraph)
+    ),
+    scopeOptions,
+    false
+  );
+}
+
+function getNegativeNodeShapeConstraints(constraints, scopeOptions) {
+  const { store, formGraph } = scopeOptions;
+  return toPathConstraints(
+    constraints.filter((constraint) =>
+      store.any(
+        constraint.object,
+        RDF("type"),
+        SHACL("NegativeNodeShape"),
```

Comment:
Is this that thing in SHACL?

Reply:
I see NotConstraintComponent?

Reply:
totally not SHACL, seems they want us to use something like this:
```
ex:NotExampleShape
	a sh:NodeShape ;
	sh:targetNode ex:InvalidInstance1 ;
	sh:not [
		a sh:PropertyShape ;
		sh:path ex:property ;
		sh:minCount 1 ;
	] .
```
which is fair, will change to that